src/test/csf/BasicNetwork.h

```diff
@@ -327,6 +165,15 @@ class BasicNetwork
     bool
     disconnect(Peer const& peer1, Peer const& peer2);
 
+    /** Check if peers are connected.
+
+        This checks that peer1 has an outbound connection to peer2.
+
+        @return `true` if peer1 is connected to peer2 and peer1 != peer2
+    */
+    bool
+    connected(Peer const& peer1, Peer const & peer2);
```

Comment:
Worth fixing since you noticed it. I must've forgotten to run clang format on this file.